app/Models/BeatmapTag.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+/**
+ * @property-read Beatmap $beatmap
+ * @property int $beatmap_id
+ * @property int $tag_id
+ * @property-read User $user
+ * @property int $user_id
+ */
+class BeatmapTag extends Model
+{
+    protected $table = 'beatmap_tags';
```

Comment:
`$table` doesn't need to be specified if the table and model already follow laravel's magic naming convention.